### PR TITLE
Feature/rad 983: Added Parallax scroll animation to job description page along with some responsive fixes

### PR DESCRIPTION
**Pull request by rogdevil**

### CODE REVIEW

File: src/components/jobDescription/jobDescriptionPage1.scss [hunk 1]
```diff
@@ -1,10 +1,36 @@
 @import "../../styles/mixins";
 @import "~sass-rem";
 
+.jobDescription1_wrapper{
+    min-height: 100%;
+    .abs_iconContainer{
+        position: absolute;
+        bottom: 0;
+        right: 0;
+        width: 624px;
```
Comment (prakhar-ira) [inline src/components/jobDescription/jobDescriptionPage1.scss:10]: use rem

File: src/components/jobDescription/jobDescriptionPage1.scss [hunk 2]
```diff
@@ -1,10 +1,36 @@
...
+        height: 624px;
+        opacity: 0.2;
+        transition: all 0.7s ease;
+
+        &.scale {
+            width: 800px;
```
Comment (prakhar-ira) [inline src/components/jobDescription/jobDescriptionPage1.scss:16]: use rem

File: src/components/jobDescription/jobDescriptionPage3.scss [hunk 3]
```diff
@@ -45,7 +50,52 @@
             gap: 12px;
             .socialIcon{
                 cursor: pointer;
+                border-radius: 24px;
+                background: var(--dark-gray-100);
+                width: 48px;
```
Comment (prakhar-ira) [inline src/components/jobDescription/jobDescriptionPage3.scss:55]: use all rem

File: src/components/jobDescription/jobDescriptionPage4.scss [hunk 4]
```diff
@@ -45,7 +50,52 @@
             gap: 12px;
             .socialIcon{
                 cursor: pointer;
+                border-radius: 24px;
+                background: var(--dark-gray-100);
+                width: 48px;
```
Comment (prakhar-ira) [inline src/components/jobDescription/jobDescriptionPage4.scss:55]: use rem

File: src/components/jobDescription/jobDescriptionPage4.scss [hunk 5]
```diff
@@ -45,7 +50,52 @@
...
+                height: 48px;
+                padding: 8px;
+                position: relative;
+                svg{
+                    z-index: 1;
+                    position: relative;
+                }
+                &::before{
+                    content: "";
```
Comment (prakhar-ira) [inline src/components/jobDescription/jobDescriptionPage4.scss:64]: isn't this css is repeated in previous component, is yes then create a common css file and import common

Comment (rogdevil) [reply src/components/jobDescription/jobDescriptionPage4.scss:64]: done

File: src/components/jobDescription/jobDescriptionPage4.scss (hunk 4 above)
Comment (rogdevil) [reply src/components/jobDescription/jobDescriptionPage4.scss:55]: done

File: src/components/jobDescription/jobDescriptionPage3.scss (hunk 3 above)
Comment (rogdevil) [reply src/components/jobDescription/jobDescriptionPage3.scss:55]: done

File: src/components/jobDescription/jobDescriptionPage1.scss (hunk 2 above)
Comment (rogdevil) [reply src/components/jobDescription/jobDescriptionPage1.scss:16]: done

File: src/components/jobDescription/jobDescriptionPage1.scss (hunk 1 above)
Comment (rogdevil) [reply src/components/jobDescription/jobDescriptionPage1.scss:10]: done

**Comment by rogdevil** [comment]
closing this pull request because it's not showing my latest commit

**Pull request closed without merging**